### PR TITLE
[7.14] [DOCS] Clarify where to complete security steps

### LINKED ISSUE #76055: [Docs] Setting up security - clarify which steps need to be performed on every node
Our new [Setting up Security](https://www.elastic.co/guide/en/elasticsearch/reference/current/security-minimal-setup.html) docs aren't always clear about which steps need to be taken on every node, and which steps can be performed once.

I did a quick pass, and I think we need:

*Every node*:

> Add the xpack.security.enabled setting to the ES_PATH_CONF/elasticsearch.yml file and set the value to true _(minimal security page)_

> Start Elasticsearch. _(minimal security page)_
 
> Restart Elasticsearch _(basic security page)_

> Add the password for your private key to the secure settings in Elasticsearch. _(https page)_

*One node only*

> Use the elasticsearch-certutil tool to generate a CA for your cluster _(basic security page)_

> Generate a certificate and private key for your node  _(basic security page)_

⬆️  _We may want to adjust this sentence a little, since it talks about "your node", but the steps we propose are to use a single cert across all nodes._

> From the directory where you installed Elasticsearch, run the Elasticsearch HTTP certificate tool  _(https page)_

See: https://discuss.elastic.co/t/setting-up-security-and-alerting-in-basic-license-cluster/280296

### PR DESCRIPTION
**Pull request by lockewritesdocs**
Updates our documentation for [Configuring security](https://www.elastic.co/guide/en/elasticsearch/reference/current/configuring-stack-security.html) to indicate which steps need to be taken on every node, and which steps can be performed once on any single node.

Preview link: https://elasticsearch_76120.docs-preview.app.elstc.co/guide/en/elasticsearch/reference/master/configuring-stack-security.html

Closes #76055

### CODE REVIEW

File: x-pack/docs/en/security/securing-communications/security-basic-setup-https.asciidoc [hunk 1]
```diff
@@ -117,25 +117,27 @@ List every hostname and variant used to connect to your cluster over HTTPS.
...
+  .. Copy the relevant `http.p12` certificate to the `$ES_PATH_CONF` directory.
+
+  .. Edit the `elasticsearch.yml` file to enable HTTPS security and specify the
+  location of the `http.p12` security certificate.
 +
 [source,yaml]
 ----
 xpack.security.http.ssl.enabled: true
-xpack.security.http.ssl.keystore.path: http.p12
+xpack.security.http.ssl.keystore.path: config/http.p12
```
Comment (ywangd) [inline x-pack/docs/en/security/securing-communications/security-basic-setup-https.asciidoc:130]: The `config/` part is not needed. Configuration files are resolved relative to the config directory.

Comment (lockewritesdocs) [reply x-pack/docs/en/security/securing-communications/security-basic-setup-https.asciidoc:130]: Thanks @ywangd -- I'll remove `config/` from the path.

File: x-pack/docs/en/security/securing-communications/security-basic-setup-https.asciidoc [hunk 2]
```diff
@@ -486,8 +488,8 @@ separate monitoring cluster prevents production cluster outages from impacting
 your ability to access your monitoring data. It also prevents monitoring
 activities from impacting the performance of your production cluster.
 
-. From the directory where you installed Elasticsearch, navigate to the
-`/kibana` directory that you created when <<encrypt-http-communication,encrypting HTTP client communications for {es}>>.
+. On any single node, navigate to the `/kibana` directory that you created when
```
Comment (tvernum) [inline x-pack/docs/en/security/securing-communications/security-basic-setup-https.asciidoc:491]: Should this be `On the node on which you generated the HTTP certificates` ? (or neater wording to that effect)

I don't think we would assume that they copied the `kibana` directory to every node (though they might have)

**Review by tvernum — APPROVED** [review]

**Comment by lockewritesdocs** [comment]
@elasticmachine run elasticsearch-ci/docs

**Pull request merged**